source/includes/_endpoint-dataset.md

```diff
@@ -130,6 +139,170 @@ is_published | boolean | true | Indicates if the dataset is published to viewers
     dataset will be the maximum of all the permissions granted.
 </aside>
 
+#### Search
+
+You can perform a cross-dataset search of dataset metadata (including variables) via the search endpoint.
+This search will return associated variables, groups, and dataset metadata.  A query string, along with filtering
+properties can be provided to the search endpoint in order to refine the results.  The query string provided is only
+used in plain-text format, any non-text or numeric characters are ignored at this time.
+
+<aside class="notice">
+
+Results are limited only to those
+datasets the user has access to.  Offset and limit parameters are also provided in order to provide performance-chunking
+options.  Variable metadata is returned with these limits based on crunch's internal index, but are filtered to include
+only variable results that match the given search term.  This means that there may be less variable search results
+provided than the limit provided by the user.
+</aside>  
+
+Here are the parameters that can be passed to the search endpoint.
+
+Paremeter             | Type        | Description
+----------------------|-------------|------------------------------------------------
+q                     | string      | query string (text and numeric only) 
```

Comment:
What does "text and numeric only" mean? What is forbidden? (We do search on underscores and stuff now, right?)